lib/token-assert.js

```diff
@@ -38,6 +38,21 @@ TokenAssert.prototype.whitespaceBetween = function(options) {
                 column: token.loc.end.column
             });
         }
+    } else if (options.hasOwnProperty('maximumSpaces')) {
```

Comment:
There is a possible trap if we ever start to use `tabs` inside of `whitespaceBetween` but I will move to `atMost` for now. I am going to add a comment on the other PR about renaming `exactly` to `lines` for consistency with `whitespaceBetween`.